pkg/framework/aries/framework.go

```diff
@@ -208,7 +219,23 @@ func WithProtocols(protocolSvcCreator ...api.ProtocolSvcCreator) Option {
 // WithLegacyKMS injects a LegacyKMS service to the Aries framework.
 func WithLegacyKMS(k api.KMSCreator) Option {
 	return func(opts *Aries) error {
-		opts.kmsCreator = k
+		opts.legacyKMSCreator = k
+		return nil
+	}
+}
+
+// WithPassphrase injects a passphrase used to secure the master key used in a SecretLock service
+func WithPassphrase(s string) Option {
```

Comment:
I don't think this shouldn't be top level - you should be providing your SecretLock. There is a particular SecretLock that is constructed with a passphrase and secret.

Reply:
function removed